javers-persistence-sql/src/main/java/org/javers/repository/sql/JaversSqlRepository.java

```diff
@@ -46,6 +47,11 @@ public JaversSqlRepository(PolyJDBC polyJDBC, CommitMetadataRepository commitRep
         this.sqlRepositoryConfiguration = sqlRepositoryConfiguration;
     }
 
+    public JaversSqlRepository withSchemaManager(SchemaNameAware schemaManager){
```

Comment:
the issue https://github.com/javers/javers/issues/542 is about two things:

- flag to disable automatic schema creation
- method to call schema creation programatically

So please implement:
- a flag method - `SqlRepositoryBuilder.withSchemaManagementEnabled(boolean)`

`JaversSqlRepository.ensureSchema()` is already here, so the second point is already done

Reply:
I mean, we don't want CustomSchemaManager

Reply:
Ok, 
so do you suggest to check if we want schema management with a flag and if it is true then invoke ensureSchema()?

Reply:
yes, and the flag should be set to true by default. Also, this flag should be available in Spring boot starters. see https://javers.org/documentation/spring-boot-integration/#javers-configuration-properties

Reply:
I implemented the feature but only tested manually ( changing default value of JaversProperties to false and see the exception). Could you give me a clue on how to test it with spring boot?